app/views/transactions/_price_break_down.haml

```diff
@@ -2,18 +2,27 @@
   - if booking
     .initiate-transaction-per-unit-wrapper
       %span.initiate-transaction-per-unit-label
-        = t("transactions.initiate.price_per_day")
+        - if unit_type == :day
+          = t("transactions.initiate.price_per_day")
+        - else
+          = t("transactions.initiate.price_per_night")
```

Comment:
Just a note for the future that label would be controlled by unit_label in the upcoming design, not pricing_unit